### DISCUSSION TITLE
A virus for the BBC Micro - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#34] [General]
# A virus for the BBC Micro - Alan Pope's blog

About a year ago, I left a comment on a Nostalgia Nerd video about Viruses. It’s a good video, worth a watch, like most of their content.
Here’s my silly comment.
At 1.7K 👍, it’s my most upvoted comment on YouTube, ever. I do enjoy free Internet points.
Some of the replies to me on YouTube were quite 🌟fun🌟.
“Any chance you made a back up? I’d love to look at the source code”

https://popey.com/blog/2023/09/a-virus-for-the-bbc-micro/

**Comment by eythian** [comment]
Ah fun. This reminds me of a story. 

I did a similar thing on the Acorns in high school (early-mid-90s I think.) I had a virus (I wasn't clever enough to write it myself from scratch, somehow I got a copy of one) and learnt how it worked, which wasn't too hard as it was written in BASIC, and made my own modifications - I forget what, random messages most likely. It worked  a similar way, basically replacing the !Boot, which was inside !App directories (similar to apple's app folders), loading itself into RAM and then loading a copy of the original. !Boot was run whenever you viewed an application and was used to set file associations and icons and such, so you didn't actually need to launch the infected application, just open a disk or directory containing it. 

Whenever it saw another !App directory, it stuck itself inside. This would happily infect any disk with an application on it inserted by the next student. However, the hard drives on these machines were locked so it couldn't infect them. Until I discovered the secret key combination at boot that would unlock it until rebooted again. Then I'd unlock the drive, load the virus from my disk, infect a few applications, and relock it again. 

I ended up leaving a disk behind in the lab with the "pristine" copy on it, and my name written on the disk. My disks were also confiscated and I was banned from the lab for the rest of the school year (but it was September so only a month or two.)

The silver lining was that the computer teacher saw I was interested in this stuff and so for the remaining years gave me access to any resources he could that I could find useful. And when you have that sort of access, you don't want to screw it up.

**Comment by youaregoddotorg** [comment]
Wow I can't believe I happened across this. I'm super flattered. Alan Pope I'm one of your favorite comments. Aaron Pope Turner. I'm the one who commented that one you found to be fun. We need to get to know each other. I have a feeling you have some age on me, but I think you and I could become great friends. Plus, I'm currently building a website about all human beings truly being God. I.E. the electricity we harness in each of us is from the same source. Most refer to it as a soul. I know it's our higher selves and every human from the beginning of time to the end of time is the exact same soul in turn being the exact same person, and at the exact same time, in a higher dimensional state. I know all this because I died about 5 years ago. Not much longer after I left you that comment. I was dead for over 8 minutes. They called my time off death. Through on a toe tag and had me on the way downstairs via the elevator going to the morgue section of the hospital. It was then I sat up looking like a ghost on Halloween night shouting out "I am God" "We are all God". "I was given the secrets to everything man had ever pondered". Are you interested in assisting me in such a sight with the mission of enlightening mankind to their true selves so that they can finally wake up until enough have that we may all assend to the 5th and higher dimensions. It's a temporal restriction due to lack of enlightened mindsets that we have not yet taken that step. I wish I could DM you my contact info.

**Reply by youaregoddotorg** [reply]
Site* or sight. I feel both work

**Reply by youaregoddotorg** [reply]
I'm restarted sometimes and speak before I do any research. I'll shoot you an email in a couple mins

**Reply by youaregoddotorg** [reply]
Retarded*
